app/views/admin/timeline/_step.html.slim

```diff
@@ -28,40 +28,46 @@
   td[
     class="#{classes} step-date step-start-date"
     data-id=entry.id
-    data-precedent-id=entry.schedule_parent_id]
+    data-precedent-id=entry.schedule_parent_id
+    data-action="show"]
     = ldate(entry.display_start_date, format: :short)
     - if entry.schedule_parent_id
       i.fa.fa-link
   td[
     class="#{classes} step-date step-end-date"
     data-id=entry.id
     data-precedent-id=entry.schedule_parent_id
-    data-dependent-step-start-date=ldate(entry.display_end_date)]
+    data-dependent-step-start-date=ldate(entry.display_end_date)
+    data-action="show"]
     = ldate(entry.display_end_date, format: :short)
     - unless entry.schedule_children.empty?
       i.fa.fa-link
 
-  td class=classes data-id=entry.id
+  td class=classes data-id=entry.id data-action="show"
     - if entry.display_duration_days && entry.display_duration_days > 0
       = t("project_step.duration_days", duration: entry.display_duration_days)
 
   - if entry.is_finalized
-    td class=classes data-id=entry.id style="background: #{entry.timeline_background_color};"
+    td class=classes data-id=entry.id style="background: #{entry.timeline_background_color};" data-action="show"
       = project_step_status(entry)
   - else
-    td class=classes data-id=entry.id
+    td class=classes data-id=entry.id data-action="show"
       = project_step_status(entry)
 
   - latest_logs = entry.latest_logs
   - logs_available = latest_logs.length
-  td.recent-logs class="#{classes} #{logs_available > 0 ? 'has-logs' : '' }" data-id=entry.id
+  td.recent-logs class="#{classes} #{logs_available > 0 ? 'has-logs' : ''}" data-id=entry.id data-action="show"
```

Comment:
I like the approach of using data-action="show"

Reply:
I noticed when there are logs you can't open the modal by clicking on the cell only. I will add this to the backlog for a fix.